content/5_Plat/DP_Bitmasks.mdx

```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
 
 ## Solution
 
-<IncompleteSection />
+<Spoiler title =  "Solution">
+
+Let our dp states be defined as $dp[S][i]$ being the number of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
```

Comment:
Why not the following? - Let $dp[S][i]$ be the number of routes that visit all the cities in the subset $S$ and end at city $i$.